Mathlib/RingTheory/TensorProduct.lean

```diff
@@ -11,7 +11,6 @@ Authors: Scott Morrison, Johan Commelin
 import Mathlib.LinearAlgebra.FiniteDimensional
 import Mathlib.RingTheory.Adjoin.Basic
 import Mathlib.LinearAlgebra.DirectSum.Finsupp
-import Mathlib.Tactic
```

Comment:
You'll need the other line from #4831 for this to pass CI

Reply:
Ok cool, I'll probably wait for that to be merged and then merge master then.